README.md

```diff
@@ -1,9 +1,88 @@
 # Reactive GraphQL
+> GraphQL reactive executor
+
+[![npm version](https://badge.fury.io/js/reactive-graphql.svg)](https://badge.fury.io/js/reactive-graphql)
+
+Execute GraphQL queries against reactive resolvers (resolvers that return Observable) to get a reactive results.
 
 This project aims to become a complete GraphQL implementation based around [RxJS](https://github.com/ReactiveX/rxjs).
 
-## Usage Example [![Edit](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/DanielMSchmidt/reactive-graphql-demo/tree/master/?hidenavigation=1)
+## Install
+```
+$ npm i reactive-graphql --save
+```
+
+## Usage
+The usage is very similar to `graphql-js`'s [`graphql`](https://graphql.org/graphql-js/graphql/#graphql) function, except that:
+- resolvers can return an Observable
+- the returned value is an Observable
 
+```js
```

Comment:
I thought it might be easier to understand to start with the simplest example possible.

Reply:
That's a good idea 👍